draft-ietf-oauth-identity-chaining.md

```diff
@@ -155,6 +168,7 @@ audience
 
 * If the request itself is not valid or if the given resource or audience are unknown, or are unacceptable based on policy, the authorization server MUST deny the request.
 * The authorization server MAY add, remove or change claims. See [Claims transcription](#claims-transcription).
+* If the inbound token is sender constrained and the resource server is acting as the Client, the Client MUST provide proof of possession of a private key, using either mTLS or DPoP, in the token exchange request to AS-A in Step (B). This is not applicable to the authorization server acting as the Client since it performs an "internal token exchange".
```

Comment:
"of its private key.."
Add to security considerations: why?: for consistency of end-to-end sender constrained tokens
"Step (B)" to maintain the practice of sender constraining tokens throughout, or something like that